studio/next.config.js

```diff
@@ -177,6 +177,16 @@ const nextConfig = {
         destination: '/project/:ref/sql/new',
         permanent: true,
       },
+      ...(process.env.NEXT_PUBLIC_BASE_PATH?.length
+        ? [
+            {
+              source: '/',
+              destination: process.env.NEXT_PUBLIC_BASE_PATH,
+              basePath: false,
+              permanent: true,
+            },
+          ]
+        : []),
```

Comment:
You can safely spread it without fallback as well, but not sure what's preferred style in codebases yet :)

```suggestion
      ...(process.env.NEXT_PUBLIC_BASE_PATH?.length && [
        {
          source: '/',
          destination: process.env.NEXT_PUBLIC_BASE_PATH,
          basePath: false,
          permanent: true,
        }
      ]),
```

Reply:
What the heck is next doing there again o_O Spreading of _any_ value inside the object is fine by the JS spec, as it's syntax sugar for `Object.assign`. Oh well ¯\\_(ツ)_/¯

Reply:
It's a spread inside an array though - maybe that's why?

Reply:
Also looking forward for some juicy syntax and TS tips in the future from you! :D

Reply:
Oh, it IS an array. I thought that redirects is object. Then yes, it only works for objects implementing `Symbol.iterator` and primitives don't do that. Too many brackets and my brain failed 🥲